packages/schema/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zapier-platform-schema",
-  "version": "10.1.0",
+  "version": "10.2.0",
```

Comment:
This version number should be left unchanged. The person who releases it (usually me or @xavdid) will bump it.

Reply:
Whoops! Reverting!